test/check-machines-create

```diff
@@ -81,6 +81,17 @@ class TestMachinesCreate(VirtualMachinesCase):
                                                             start_vm=False,
                                                             pixel_test_tag="auto"))
 
+        # Test OS auto detection can change "Operatin system" field without Ooops
+        runner.cancelDialogTest(TestMachinesCreate.VmDialog(self, sourceType='url',
+                                                            location=config.VALID_URL,
```

Comment:
I don't get why you want to use locationSecondInput new variable. Can't you just set directly the location to TREE_URL and let the dialog autodetect the OS?

Reply:
Yeah, make sense, I just follow the step of BZ and forget that